nixos/modules/misc/ids.nix

```diff
@@ -288,6 +288,7 @@
       kresd = 270;
       rpc = 271;
       geoip = 272;
+      slimserver = 273;
```

Comment:
slimserver definitely has non-transient state (settings, music library metadata). So should I use a static UID?